src/configuration/CalibrationConfig.h

```diff
@@ -89,7 +105,7 @@ namespace SlimeVR {
             float G_off[3];
         };
 
-        enum CalibrationConfigType { NONE, BMI160, MPU6050, MPU9250, ICM20948, ICM42688 };
+        enum CalibrationConfigType { NONE, BMI160, BMI323, MPU6050, MPU9250, ICM20948, ICM42688 };
```

Comment:
Wouldn't adding a new entry in the middle break the saved configs for every sensor after?

Reply:
I pushed a commit to put BMI323 at the end of the list